tests/integration-tests/tests/performance_tests/test_scaling/test_scaling/pcluster.config.yaml

```diff
@@ -0,0 +1,51 @@
+Region: {{ region }}
+Image:
+  Os: {{ os }}
+HeadNode:
+  InstanceType: c5.24xlarge
+  Networking:
+    SubnetId: {{ public_subnet_id }}
+  Ssh:
+    KeyName: {{ key_name }}
+  Iam:
+    AdditionalIamPolicies:
+      - Policy: arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore #Required to report patching status
+Scheduling:
+  Scheduler: slurm
+  SlurmQueues:
+    - Name: queue-0
+      ComputeResources:
+        - Name: compute-resource-0
+          InstanceType: {{ instance }}
+          MinCount: {{ max_nodes }}
```

Comment:
By having MinCount == max_nodes we can face bootstrap issues at cluster creation time. What about having it set to 0 and just consider the "scaling" case for dynamic node?

I'd remove queue-0 and queue-2 to avoid confusion.

Reply:
I think the goal of this test is to detect issues both on large scale static and dynamic fleets.
I suggest to keep a static fleet of 1K nodes that, in case of issues, will make the test to fail at cluster creation time.
Why you think it's not necessary?

I would remove the third queue as it is unused

Reply:
The only thing I'm thinking is that if the creation fails it's not so easy to understand the reason, so we need to check the cluster logs. Anyway I'm fine keeping it.

Reply:
[For the static nodes] A trade-off could be to start with a MinCount of `0` and run an update-cluster with `MinCount==max_nodes`.